packages/rrweb/src/replay/index.ts

```diff
@@ -148,8 +146,16 @@ export class Replayer {
 
   private newDocumentQueue: addedNodeMutation[] = [];
 
-  private mousePos: mouseMovePos | null = null;
-  private touchActive: boolean | null = null;
+  // Map of touch ID to the unique vars used to show gestures
+  private multiTouchMap: Record<
+    number,
+    {
+      touchActive: boolean | null;
+      mouse: HTMLDivElement;
```

Comment:
I would go ahead and rename this to `pointerEl` as well for similar reasons + to indicate it's an element

Reply:
i feel like with all these data structure changes, we might need to update places in the `sentry` repo where we're referencing things like `replayer.mouse` if they exist